modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/impl/ExpBucket.java

```diff
@@ -0,0 +1,23 @@
+package com.intuit.wasabi.repository.cassandra.impl;
+
+
+import com.intuit.wasabi.experimentobjects.Experiment;
+import com.intuit.wasabi.experimentobjects.Bucket;
+
+
+class ExpBucket {
```

Comment:
@mooxter This class needs to overrides equals() & hashcode() methods, probably add Builder as well...